src/app/course-manager/authors-list/authors-list.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-authors-list',
+  templateUrl: './authors-list.component.html',
+  styleUrls: ['./authors-list.component.sass']
+})
+export class AuthorsListComponent implements OnInit {
+  @Input() parentForm: FormGroup;
+
+  ngOnInit() {
+    console.log(this.parentForm);
+  }
```

Comment:
This component looks like a presentation component. Then you can add `OnPush` strategy to it.